### PR TITLE
assignment 2

### PR DESCRIPTION
**Pull request by akshitmittal1**
.

### CODE REVIEW

File: assignment_2/akshit/README.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+## Problem Statement:
+
+Train a model to print the first n elements of the Fibonacci sequence starting at integers a and b. Note that a < b, and n <= 100 are user supplied arguments. You may not use the standard summing operation to print the next element.Example I/O: Input: a=5, b=7, n=5 Output: 5, 7, 12, 19, 31 Input: a=7, b=41, n=6 Output: 7, 41, 49, 90, 139, 229
```
Comment (AnirudhDagar) [inline assignment_2/akshit/README.md]: Please correct this.
a=7 b=41 n=6 Output: 7, 41, 48, 89, 137, 226

**Review by AnirudhDagar — CHANGES_REQUESTED** [review]
Looks good!
A few things though...
1. Try to submit a PR next time in a .py file, reviewing is a bit difficult in .ipynb file.
2. Also, function string comments would help. :)

File: assignment_2/akshit/README.md (hunk 1 above)
Comment (akshitmittal1) [reply assignment_2/akshit/README.md]: Thanks, and I will keep the above things in ind next time

**Review by AnirudhDagar — APPROVED** [review]

**Comment by AnirudhDagar** [comment]
Merged this :)
ps. What's up with the SVM assign?

**Comment by akshitmittal1** [comment]
Hi, working on it, will make a pull request positively by 15th



[image: Mailtrack]
<https://mailtrack.io?utm_source=gmail&utm_medium=signature&utm_campaign=signaturevirality5&>
Sender
notified by
Mailtrack
<https://mailtrack.io?utm_source=gmail&utm_medium=signature&utm_campaign=signaturevirality5&>
05/09/19,
7:34:40 PM

**Pull request merged**